index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tekle's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+      integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <div class="container">
+        <p class="logo">My Logo</p>
+        <a href="#" class="btn-menu-toggle"><i class="fas fa-bars"></i></a>
+      </div>
+    </header>
+    <section class="headline">
+      <div class="container">
+        <h2 class="headline--title">I’m Tekle Glad to see you!</h2>
```

Comment:
- [ ]  I would suggest using the `<h1>` tag instead of the `<h2>`. Remember that there could only be one `<h1>` per page.
The `<h1>` tag defines the most important heading and this text could be considered as the most important heading.

- [ ] The main title should have a header and a subtitle, yours seems to have only a header.
Your design
![por](https://user-images.githubusercontent.com/53356820/143449396-bef826ad-566f-4465-865f-a51905ccf615.png)

Figma design
![por1](https://user-images.githubusercontent.com/53356820/143449487-c3caf282-6762-4468-821b-b90a747a1afb.png)